VultisigApp/VultisigApp/Views/Components/VaultDetailBalanceContent.swift

```diff
@@ -8,8 +8,8 @@
 import SwiftUI
 
 struct VaultDetailBalanceContent: View {
-    let vault: Vault
-    
+    @ObservedObject var vault: Vault
```

Comment:
why we need to have vault as an ObservableObject? will it be changed ?

Reply:
As we're not storing rate inside coin anymore, we should observe vault to track changes. 
I'm trigger vault update with `vault.objectWillChange.send()` in CryptoPriceService